### PR TITLE
implemented

### PR DESCRIPTION
**Pull request by vshand11**
Implemented 1) CRDCDH-2009 2) CRDCDH-2060.  These two tasks has overlapping requests and highly related.

### CODE REVIEW

File: services/user.js [hunk 1]
```diff
@@ -323,23 +357,9 @@ class User {
...
-                "$match": { _id: params.organization }
-            }, {"$limit": 1}]);
-            const newOrg = result?.[0];
-
-            if (!newOrg?._id || newOrg?._id !== params.organization) {
-                throw new Error(ERROR.INVALID_ORG_ID);
-            }
-
-            updatedUser.organization = orgToUserOrg(newOrg);
-        } else if ((typeof(params.organization) !== "undefined" && !params.organization && user[0]?.organization?.orgID)) { // Data Curator should not be assigned any Org
```
Comment (knockknockyoo) [inline services/user.js:340]: Is this required for the current implementation? **The Data Curator should not be assigned any organization**. It was one of requirements in the past.

File: services/user.js [hunk 2]
```diff
@@ -353,32 +373,12 @@ class User {
...
-                 if (!approvedStudies || approvedStudies.length === 0) {
-                     throw new Error(ERROR.INVALID_NO_STUDIES);
-                 }
-                 const approvedStudyArr = approvedStudies.map((study) => study._id)
-                 if (!includesAll(approvedStudyArr, params.studies)) {
-                     throw new Error(ERROR.INVALID_NOT_APPROVED_STUDIES);
-                 }
-            }
-            updatedUser.studies = params?.studies;
+        const valid_studies = await this.#findApprovedStudies(params.studies);
```
Comment (knockknockyoo) [inline services/user.js]: Please, change this name convention.

**Review by knockknockyoo — CHANGES_REQUESTED** [review]

File: services/user.js (hunk 2 above)
Comment (vshand11) [reply services/user.js]: Updated.

**Comment by knockknockyoo** [comment]
studies: [String] is optional in the graphql scheme. If **studies** parameter is not included, it will cause unexpected error. We should only store **studies** only if a studies parameter is included.

**Review by AustinSMueller — APPROVED** [review]

**Pull request merged**